styleguide/global/voice.jade

```diff
@@ -16,7 +16,7 @@
 hr
 
 .row.half-padding-top
-  h5 How to write to our Templates
+  h5 How to write to Ecommerce Templates
   p
     a(href="https://raxglobal.sharepoint.com/Global%20Ecommerce/Shared%20Documents/Forms/AllItems.aspx?id=%2FGlobal%20Ecommerce%2FShared%20Documents%2FeComm%20Ops%2FWeb%20page%20changes%2FTemplate%20Documentation") Writing Templates
```

Comment:
Let's mask this URL. You can use https://rax.io/create to create shortened/obfuscated URLs that only resolve in-network. Wouldn't this be the same URL as in other places? It could probably be centralized as a global variable - http://harpjs.com/docs/development/globals